app/services/create_new_device_alert.rb

```diff
@@ -9,7 +9,7 @@ def perform(now)
       sql_query_for_users_with_new_device,
       tvalue: now - IdentityConfig.store.new_device_alert_delay_in_minutes.minutes,
     ).each do |user|
-      emails_sent += 1 if clear_new_device_and_send_email(user)
+      emails_sent += 1 if lapse_sign_in_notification_window_and_send_alert(user)
```

Comment:
I was on the fence between the word "lapse" and "expire". I think "expire" is a little more plaintext, but "lapse" feels a little closer to the specific mechanic of having a valid period of time to complete the task.

Reply:
Discussed with @myfriendgu and settled on new language in 95833a8 using "timeframe expired" terminology.